ONSdigital/eq-author
# DIGEQ-115 Validation ui

**What**
This is @alexmorris UI design for the validation rules.

**How to test**
1. Create a new survey
2. Add a text question and a numeric question.
3. Observe the new styling

**Who can review**
Anyone apart from @warren-methods and @alexmorris
